litellm/llms/notdiamond.py

```diff
@@ -0,0 +1,253 @@
+import types
+import requests
+from typing import Callable, Optional, Dict, List
+import httpx
+
+import litellm
+from litellm.utils import ModelResponse
+from litellm._version import version
+
+
+# dict to map notdiamond providers and models to litellm providers and models
+notdiamond2litellm = {
```

Comment:
Since this is a globally-scoped variable I would suggest using the `UPPERCASE` [convention](https://github.com/kengz/python?tab=readme-ov-file#references--prefer-const)